packages/components/src/toggle-control/index.native.js

```diff
@@ -11,15 +16,16 @@ const ToggleControl = memo(
 	( { label, checked, help, instanceId, className, onChange, ...props } ) => {
 		const id = `inspector-toggle-control-${ instanceId }`;
 
+		const helpLabel = help && isFunction( help ) ? help( checked ) : help;
+
 		return (
 			<SwitchCell
 				label={ label }
 				id={ id }
-				help={ help }
+				help={ helpLabel }
```

Comment:
I noticed that on Android when tapping over the "Resize for smaller devices" element the ripple animation covers also the help message, not sure if this is expected 🤔 .


https://user-images.githubusercontent.com/14905380/129365367-ba8650fa-0e8a-4167-b9c9-abfbfe6a4785.mp4

Reply:
I think I didn't make changes around that in this PR. I've checked with Audio Block settings and it seems to be the same there. Pressing directly on the toggle doesn't create that effect, so I guess it could be the bottom sheet row being a touchable that's making that happen.

Reply:
Right, looks like this is related to the fact that the help message is rendered within the `TouchableRipple` component:
https://github.com/WordPress/gutenberg/blob/b77066b38ae7e7a8894b33f328ccfeedfd265c8f/packages/components/src/mobile/bottom-sheet/cell.native.js#L397-L401

If we're using it in other places I guess it's expected, thanks for checking it 🙇 .